<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoErrorGroupItem</class>
 <widget class="QWidget" name="NoErrorGroupItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>51</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-bottom: 1px solid #E2E4E5;
background-color: white;
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="groupIconLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="groupNameLabel">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Название группы</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="abortButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-color: #e2e9ef;
color: #e64646;
border-radius: 4px;
padding: 7px 16px 8px;
}

QPushButton:hover
{
	background-color: #dfe6ed;
	text-decoration: underline;
}

QPushButton:pressed
{
	color: #e2e9ef;
	text-decoration: none;
	background-color: #e64646;
}</string>
     </property>
     <property name="text">
      <string>Аборт поста</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
